be/src/fs/hdfs/hdfs_fs_cache.h

```diff
@@ -48,7 +52,11 @@ class HdfsFsCache {
 
 private:
     std::mutex _lock;
-    HdfsFsMap _cache;
+    int _cur_client_idx{0};
+    constexpr static int _max_cache_clients = 8;
```

Comment:
Only cache 8 clients?

Reply:
This number is copied from `fs_s3.cpp`, 8 clients are enough.